javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -30,6 +33,21 @@
 public final class ModifierSet {
 
     /* Definitions of the bits in the modifiers field.  */
+    
+    private static Map<javax.lang.model.element.Modifier, Integer> ENUM_TO_INT = new HashMap<javax.lang.model.element.Modifier, Integer>();
```

Comment:
Please no ref to jx.l.m in this. How do we support j8's 'default' if we build on j6 ?